types/faker-schema/index.d.ts

```diff
@@ -0,0 +1,15 @@
+// Type definitions for faker-schema 1.0
+// Project: https://github.com/knicklabs/faker-schema
+// Definitions by: Yury A Troynov <https://github.com/yutro>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export function withProbability<T>(value: (() => T) | T, probability?: number): <S>(schema: S) => (T extends () => void ? S : T) | null;
+
+export class Schema<T> {
+    private seed?: number;
+    private blueprint: () => T;
```

Comment:
those are private and are overriden via setters only, am I correct? (desribed in readme)

Reply:
So there are no classical setter for `seed `, its being overridden only when makeOne method called with seed parameter.
ex.
https://github.com/knicklabs/faker-schema/blob/f953b3a6011c288a3743740cef5eb116ff50c12b/src/lib/Schema.js#L6
line 25.